common/scripted_effects/wc_horde_invasion_effects.txt

```diff
@@ -45,28 +45,93 @@ spawn_horde_troops_effect = {
 			inheritable = no
 			name = horde_event_troops
 			levies = {
-				value = 1250
+				value = 1000
 			}
 			men_at_arms = {
-				type = light_footmen
-				stacks = 3
+				type = savage
+				stacks = 1
 			}
 			men_at_arms = {
-				type = light_horsemen
+				type = blackrock_legionnaire
 				stacks = 1
 			}
 			men_at_arms = {
-				type = armored_horsemen
-				stacks = 1
+				type = wolfrider
+				stacks = 2
 			}
 			men_at_arms = {
-				type = mangonel
-				stacks = 1
+				type = trebuchet
+				stacks = 10
+			}
+			location = capital_province
+		}
+	}
+}
+spawn_orc_savage_troops_effect = {
+	if = {
+		limit = { exists = capital_province }
+		spawn_army = {
+			uses_supply = no
+			inheritable = no
```

Comment:
why not inheritable? would be nice, if they at least didn't disappear into thin air before the 2. war is done...

Reply:
Every Warchief creates a new army so it's not inheritable.

Reply:
Ok then :)